phase5/Solution1/ConsoleApp1/Runnable.cs

```diff
@@ -0,0 +1,58 @@
+using SampleLibrary;
+
+namespace ConsoleApp1;
```

Comment:
naming convention (its meaningless)

Reply:
this concerns phase 6
we will fix it there